test/test_lswt.jl

```diff
@@ -43,7 +43,7 @@
     @assert energy_per_site(sys) ≈ -328.38255
 
     # Verify that this is a local minimum of energy
-    @test norm(Sunny.proj.(Sunny.energy_grad_coherents(sys), sys.coherents)) < 1e-8
+    @test norm(Sunny.proj.(Sunny.energy_grad_coherents(sys), sys.coherents)) < 2e-8
```

Comment:
On my Mac, I verified we can get many more digits of accuracy here by selecting, say, `x_tol=1e-16` inside `minimize_energy!`.